fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/Statistics.java

```diff
@@ -32,17 +32,23 @@ public double getOutputRowCount() {
 
     public double getOutputSize(ColumnRefSet outputColumns) {
         double totalSize = 0;
+        boolean nonEmpty = false;
         for (Map.Entry<ColumnRefOperator, ColumnStatistic> entry : columnStatistics.entrySet()) {
             if (outputColumns.contains(entry.getKey().getId())) {
                 totalSize += entry.getValue().getAverageRowSize();
+                nonEmpty = true;
             }
         }
+        if (nonEmpty) {
```

Comment:
outputColumns not in columnStatistics? May be the outputColumns and column statistics were not calculated correctly

Reply:
Just for the sake of conservation